From 80db650848f75e8169c6721904e7249c35e8e924 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Wed, 17 May 2023 18:43:56 +0800
Subject: [PATCH 2/2] amlbian: Optimize logs [1/1]

PD#SWPL-111046

Problem:
Optimize logs.

Solution:
Optimize logs.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: Iaaf6d7e45206fc7c2e787cb68e63781c34aceb17
---
 cmd/part.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/part.c b/cmd/part.c
index bfb6488b0f2..cb32bd4feb4 100644
--- a/cmd/part.c
+++ b/cmd/part.c
@@ -95,7 +95,7 @@ static int do_part_list(int argc, char * const argv[])
 			int r = part_get_info(desc, p, &info);
 
 			if (r != 0)
-				continue;
+				break;
 
 			if (bootable && !info.bootable)
 				continue;
-- 
2.25.1

